tool/plugin/lib/plugin.dart

```diff
@@ -515,6 +480,9 @@ class BuildCommand extends ProductCommand {
   final BuildCommandRunner runner;
 
   BuildCommand(this.runner) : super('build') {
+    argParser.addOption('only-version',
```

Comment:
This could also be useful for debugging. Adding an 'abbr' would be handy, but not urgent.

Reply:
sounds good, will add an abbreviation

Reply:
(`-o`)

Reply:
LGTM